release-notes/7.0/preview/api-diff/preview4/Microsoft.WindowsDesktop.App/7.0-preview4_System.Windows.Forms.md

```diff
@@ -0,0 +1,221 @@
+-            bool System.Collections.ICollection.IsSynchronized { get; }
++            bool ICollection.IsSynchronized { get; }
+-            object System.Collections.ICollection.SyncRoot { get; }
++            object ICollection.SyncRoot { get; }
+-            bool System.Collections.IList.IsFixedSize { get; }
++            bool IList.IsFixedSize { get; }
+-            object System.Collections.IList.this[int index] { get; set; }
++            object? IList.this[int index] { get; set; }
+-            int System.Collections.IList.Add(object value);
++            int System.Collections.IList.Add(object? value);
+-            bool System.Collections.IList.Contains(object index);
++            bool System.Collections.IList.Contains(object? index);
+-            int System.Collections.IList.IndexOf(object index);
++            int System.Collections.IList.IndexOf(object? index);
+-            void System.Collections.IList.Insert(int index, object value);
++            void System.Collections.IList.Insert(int index, object? value);
+-            void System.Collections.IList.Remove(object value);
++            void System.Collections.IList.Remove(object? value);
+         }
+         public class CheckedItemCollection : ICollection, IEnumerable, IList {
+-            bool System.Collections.ICollection.IsSynchronized { get; }
++            bool ICollection.IsSynchronized { get; }
+-            object System.Collections.ICollection.SyncRoot { get; }
++            object ICollection.SyncRoot { get; }
+-            bool System.Collections.IList.IsFixedSize { get; }
++            bool IList.IsFixedSize { get; }
+-            public object this[int index] { get; set; }
++            public object? this[int index] { get; set; }
+-            public bool Contains(object item);
++            public bool Contains(object? item);
+-            public int IndexOf(object item);
++            public int IndexOf(object? item);
+-            int System.Collections.IList.Add(object value);
++            int System.Collections.IList.Add(object? value);
+-            void System.Collections.IList.Insert(int index, object value);
++            void System.Collections.IList.Insert(int index, object? value);
+-            void System.Collections.IList.Remove(object value);
++            void System.Collections.IList.Remove(object? value);
+         }
+     }
+     public class FontDialog : CommonDialog {
+-        public event EventHandler Apply;
++        public event EventHandler? Apply;
+     }
+     public class ListView : Control {
+         public class CheckedIndexCollection : ICollection, IEnumerable, IList {
+-            bool System.Collections.ICollection.IsSynchronized { get; }
++            bool ICollection.IsSynchronized { get; }
+-            object System.Collections.ICollection.SyncRoot { get; }
++            object ICollection.SyncRoot { get; }
+-            bool System.Collections.IList.IsFixedSize { get; }
++            bool IList.IsFixedSize { get; }
```

Comment:
Is this expected? Looks like noise...

Reply:
in one of the last iterations, i remember tool was updated to exclude these kind of changes. @carlossanlop, anything changed in tool since then?

Reply:
Something changed in Microsoft.Cci.Extensions because I made sure to run the tool with the fix. I'll have to investigate.

Reply:
Fixed. I had some fixes in my local machine that I merged incorrectly with the latest bits in main, causing the EIIs to show up again with the full name.